tests/scripts/test_scripts.py

```diff
@@ -563,60 +500,6 @@ def test_transfer_learning(tmpdir: str) -> None:
     _check_rollout_stats(run.result)
 
 
-@pytest.mark.parametrize(
-    "named_configs",
-    (
-        [],
-        ["reward.reward_ensemble"],
-    ),
-)
-def test_preference_comparisons_transfer_learning(
```

Comment:
Why are we deleting this test?

Reply:
This was an accident. Thanks for pointing it out. Should be back now.